app/ide-desktop/lib/dashboard/src/modals/AboutModal.tsx

```diff
@@ -70,7 +70,12 @@ export default function AboutModal() {
       {/* This button is not visible - it is only to provide the button for this `DialogTrigger`. */}
       <aria.Button className="h-0 w-0" />
       <ariaComponents.Dialog className="w-[30rem]">
-        <div className="relative flex items-center gap-4">
+        <div
+          className="relative flex items-center gap-4"
+          onClick={event => {
```

Comment:
Oh, I guess we should fix the root cause instead. Can you elaborate once more on why we have to add `stopPropogation` here?

Reply:
if we click the background, the dialog closes for some reason

Reply:
i guess it was to stop modals from closing when trying to select a range in an input and overshooting.

some basic testing indicates that there's code in `Modal.tsx` which seems to do the same thing for dialogs that still use `Modal`, so it seems like it can safely be removed